features/support/env.rb

```diff
@@ -18,5 +16,29 @@ def copy_package
 end
 
 Before do
+  # This is a bit of a hack to let us record the last command's exit code for
+  # asserting against. This module redefines 'run_docker_compose_command' and then
+  # gets called before the real 'run_docker_compose_command' because it is
+  # prepended (see Module#prepend). This lets us record the last exit code, which
+  # should always be the actual command we run in the tests
+  # There is possibility for tests to interfere with each other, but only if they
+  # don't run commands themselves. In which case, why would they care about the
+  # last exit code??
+  module RecordExitCode
+    attr_reader :last_exit_code
+
+    def run_docker_compose_command(...)
+      raise "You can't use RecordExitCode without a super class!" unless defined?(super)
+
+      _output, @last_exit_code = super(...)
```

Comment:
This seems sensible, I'll add this into the Docker class and appropriate steps to access it.

New steps:
- Then the exit code of the last docker command was successful
- Then the exit code of the last docker command was {Int}

Reply:
I don't think that's necessary; presumably at some point we'll be replacing this with the new interactive CLI steps? This just exists so that we can use those assertions before they're in a MR release